### DISCUSSION TITLE
What grade of GPU do I need to use ImagePredictor of autogluon?

### DISCUSSION THREAD

**Discussion by 155cannon** [autogluon/autogluon#1988] [Q&A] [answered]
1060 6G is OK?

**Comment by Innixma** [comment, marked as answer]
1060 6G is maybe just barely enough, but you'd have to try it yourself and see. For many models it will not be enough.

**Reply by 155cannon** [reply]
ImagePredictor doesn't work with GTX 1060 6G or RTX A2000 6G. Can you tell me which GPU you used for ImagePredictor?

**Reply by Innixma** [reply]
We use Nvidia Tesla T4 (equivalent to a 1060 with 16G) and NVIDIA A10G (equivalent to a 2080 with 12G). These are used in AWS g4 and g5 instances respectively, which we use to build and run our tutorials.

**Reply by 155cannon** [reply]
Thank you!
